lib/schemes/local.js

```diff
@@ -47,7 +51,9 @@ export default class LocalScheme {
         ? this.options.tokenType + ' ' + result
         : result
 
-      this.$auth.setToken(this.name, token)
+      // Change Token Name on Impersonate (only if impersonating user has 'admin' scope)
```

Comment:
this forces to use `admin` name for administrative scope, and will not work in case other scope name. What if someone will name administrative scope `admins` instead of `admin`, or `root`?